### PR TITLE
Update version number & documentation

### PR DESCRIPTION
**Pull request by stratofax**
Tracking version history on the new development branch.

### CODE REVIEW

File: VERSIONS.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# VERSIONS
```
Comment (ion-oset) [inline VERSIONS.md:1]: Convention is to call a file this "Changelog.md" or "CHANGELOG.md".

File: src/electos/ballotmaker/constants.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 import errno
 
 PROGRAM_NAME = "BallotMaker"
-VERSION = "0.1.1"
+VERSION = "0.1.2"  # incremented September 7, 2022
```
Comment (ion-oset) [inline src/electos/ballotmaker/constants.py:7]: I'm a [-0](https://www.apache.org/foundation/voting.html#expressing-votes-1-0-1-and-fractions) on this. The date status is a nice thing to see. It has the downside is that it's one more step to update when the version changes. It's slightly less bookkeeping to defer to the changelog as the source of the dates.

File: README.md [hunk 3]
```diff
@@ -14,19 +16,27 @@ This Python project uses Poetry for dependency management. To get started:
...
 ## Running Tests
 
 After you've followed the steps above in **Getting Started**, you can also run the `pytest` suite with this command:
 
 ```python
-poetry run pytest
+pytest
+```
+
+Or, if you'd like to review coverage information and logging updates, try this command:
```
Comment (ion-oset) [inline README.md:36]: Nice 👍🏼

**Review by ion-oset — COMMENTED** [review]
LGTM. A couple of comments below.

File: VERSIONS.md (hunk 1 above)
Comment (stratofax) [reply VERSIONS.md:1]: Agreed

File: src/electos/ballotmaker/constants.py (hunk 2 above)
Comment (stratofax) [reply src/electos/ballotmaker/constants.py:7]: In fact, I found the dates for the changelog by reviewing the Git history for this line. Plus, I've got a little keyboard shortcut on my Mac that types out the date. If it's a hassle I'll delete it.

**Pull request merged**